docs/concept-exercises.md

```diff
@@ -87,7 +91,7 @@ See the C# floating-point-numbers exercise's [design.md file][csharp-docs-design
 
 ### `.meta/config.json`
 
-This file contains meta information on the exercise, which currently only includes the exercise's contributors.
+This file contains meta information on the exercise, which currently includes the exercise's contributors, and language version requirements (optional).
```

Comment:
```suggestion
This file contains meta information on the exercise, which currently includes the exercise's contributors, and (optionally) language version requirements.
```

Reply:
@SaschaMann Could you add the optionally part to all changed files?